### ISSUE TITLE
Crash on large rom directory

### ISSUE THREAD

**Issue by myMartek** [RSDuck/vba-next-switch#33]
Hi, I have a directory on my switch which contains all english roms ever. When I navigate to this directory I can see some strange graphic artifacts where the "B" button and the VBA logo should be. As soon I go a directory back or start a rom the switch becomes unresponsive.

When I open a directory with less files everything works as expected.

**Comment by fennectech** [comment]
This sounds like a buffer overflow.

**Comment by RSDuck** [comment]
yes, it is one. The buffer where the directory names are stored in has a fixed size
